packages/medusa/tsconfig.json

```diff
@@ -16,7 +16,13 @@
     "noImplicitThis": true,
     "allowJs": true,
     "skipLibCheck": true,
-    "downlevelIteration": true // to use ES5 specific tooling
+    "downlevelIteration": true, // to use ES5 specific tooling
+    "baseUrl": ".",
+    "paths": {
+      "@models/*": [
+        "./src/models/*"
+      ]
+    }
```

Comment:
question: Is this a needed?

Reply:
No I will remove it and it could be part of another pr. It is to avoid infinite ../../../